mono/sgen/sgen-los.c

```diff
@@ -726,6 +799,32 @@ sgen_los_pin_object (GCObject *data)
 	sgen_binary_protocol_pin (data, (gpointer)SGEN_LOAD_VTABLE (data), sgen_safe_object_get_size (data));
 }
 
+void
+sgen_los_iterate_pin_objects (SgenGrayQueue *gray_queue, gboolean finish_concurrent_mode)
```

Comment:
This name is confusing, I thought we iterated pinned objects. It should be `sgen_los_pin_objects`

Reply:
Agree, intention wit the naming was to iterate and pin objects, but name could be simplified.